ClientKit/ClientKit.cs

```diff
@@ -30,6 +32,18 @@ namespace OSVR
 {
     namespace ClientKit
     {
+        public sealed class SafeClientContextHandle : SafeHandleZeroOrMinusOneIsInvalid
+        {
+            public SafeClientContextHandle() : base(true) { }
+
+            [ReliabilityContract(Consistency.WillNotCorruptState, Cer.MayFail)]
+            protected override bool ReleaseHandle()
+            {
+                System.Diagnostics.Debug.WriteLine("Client shutdown");

```

Comment:
Do we need to preface this with something like `[OSVR]` or does .NET does something like that automagically?